src/molecules/tanstack-table/utils.tsx

```diff
@@ -38,20 +39,49 @@ export function getCommonPinningStyles<TData>({
 
 export function tanstackTableCreateColumnsByRowData<T>(params: {
   languageData?: Record<string, string>;
+  links?: Record<string, TanstackTableColumnLink>;
   row: Record<string, string | number | boolean | Date | null>;
 }) {
-  const { row, languageData } = params;
+  function createCell(
+    accessorKey: string,
+    row: Row<T>,
+    link?: TanstackTableColumnLink
+  ) {
+    if (!link) {
+      return <div>{row.getValue(accessorKey)?.toString()}</div>;
+    }
+    let url = link.prefix;
+    if (link.targetAccessorKey) {
+      url +=
+        `/${row
+          ._getAllCellsByColumnId()
+          ?.[link.targetAccessorKey || ''].getValue()
+          ?.toString()}` || '';
+    }
+    if (link.suffix) {
+      url += `/${link.suffix}`;
+    }
+    return (
+      <a href={url} className="font-medium underline">
```

Comment:
can we don't use "a" tag by default I prefer next "Link" or we can pass as prop

Reply:
We have next in the project.

Reply:
I will change this in next pr.